asyncapi/v2/validation_test.go

```diff
@@ -83,7 +83,7 @@ func TestFromDocJsonSchemaMessageValidator(t *testing.T) {
 		t.Run(test.name, func(t *testing.T) {
 			// Doc generation
 			channel := NewChannel(t.Name())
-			channel.Subscribe = NewSubscribeOperation(&Message{PayloadField: test.schema})
+			channel.Publish = NewPublishOperation(&Message{PayloadField: test.schema})
```

Comment:
Does it make sense? 

When configuring the proxy, we should check for `publish` type operations as it acts as a proxy for messages published by clients.